packages/compass-indexes/src/modules/search-indexes.spec.ts

```diff
@@ -124,17 +123,10 @@ describe('search-indexes module', function () {
       expect(store.getState().searchIndexes.status).to.equal('REFRESHING');
     });
 
-    it('loads and sorts the indexes', async function () {
+    it('loads the indexes', async function () {
```

Comment:
The default order is now different. How do we know it still appears in the same order to the user?